clang/lib/StaticAnalyzer/Checkers/WebKit/PtrTypesSemantics.cpp

```diff
@@ -462,7 +462,7 @@ bool isPtrConversion(const FunctionDecl *F) {
   const auto FunctionName = safeGetName(F);
   if (FunctionName == "getPtr" || FunctionName == "WeakPtr" ||
       FunctionName == "dynamicDowncast" || FunctionName == "downcast" ||
-      FunctionName == "checkedDowncast" ||
+      FunctionName == "checkedDowncast" || FunctionName == "bit_cast" ||
```

Comment:
Does this check need to be replicated in `TrivialFunctionAnalysis` at line 661?

Reply:
oh, that's a good point. I think we can see the difference in local variable checker.

Reply:
Fixed!